sabnzbd/downloader.py

```diff
@@ -477,6 +485,18 @@ def sleep_time_set(self):
         self.sleep_time = cfg.downloader_sleep_time() * 0.0001
         logging.debug("Sleep time: %f seconds", self.sleep_time)
 
+    def recv_threads_set(self):
```

Comment:
Why do we need this?
No need to add support for live-changes for Special's settings.

Reply:
I wasn't sure. I'll remove it.